### PR TITLE
Text inputs version 2 updates

### LINKED ISSUE #291: Update Text Inputs layout/design to v2
**Description**
This card is a part of a series of cards to update the latest style of the text inputs to version 2. Below are the updated visual designs for reference:

inputs_text_inputs_ui_v2

### PR DESCRIPTION
**Pull request by juliaelman**
This pull request is for the updated designs for text input fields. It is also downstream from PR #301 and utilizes the variables inside of that work.

Fixes #291

### CODE REVIEW

**Comment by maya** [comment]
@juliaelman the design uses a gray border in focus state: 
screen shot 2015-08-17 at 9 03 39 am

Yours uses the blue border:
screen shot 2015-08-17 at 9 02 38 am

Can you fix this?

**Comment by maya** [comment]
Label text should be normal weight via the new designs.

**Comment by maya** [comment]
Where is this error state?

screen shot 2015-08-17 at 9 06 22 am

**Comment by juliaelman** [comment]
@maya updates made and thank you for your (as always) helpful review! Let me know if it looks ok and/or if you have any other changes.

**Comment by maya** [comment]
@juliaelman can we add in the left vertical red border like the designs have?

**Comment by juliaelman** [comment]
@maya added red border here bd5fac20bc025e27101f61a7da2322adf93d11dc

**Comment by maya** [comment]
@juliaelman can you make the left border 4px? I checked and that's the width in the designs. Slightly thicker than the 3px input borders.

File: _elements/inputs.md [hunk 1]
```diff
@@ -19,19 +19,25 @@ title: Inputs
...
   <label for="input-focus">Text Input Focused</label>
   <input class="usa-input-focus" id="input-focus" name="input-focus" type="text">
 
-  <label for="input-error">Text Input Error</label>
-  <input class="usa-input-error" id="input-error" name="input-error" type="text">
+  <div class="input-error">
+    <label for="input-error">Text Input Error</label>
+    <span class="input-error_message" role="alert">Helpful error message</span>
+    <input class="usa-input-error" id="input-error"  aria-describedby="input-error-message" name="input-error" type="text">
+  </div>
```
Comment (maya) [inline _elements/inputs.md]: Does this mean we should add a wrapping div around all inputs (label + input) to keep this consistent and able to trigger an error message?

**Comment by juliaelman** [comment]
@maya completed here 7785d16

File: _elements/inputs.md (hunk 1 above)
Comment (juliaelman) [inline _elements/inputs.md]: Possibly, yes! For now, I would recommend that we focus on implementing this work and add a card in the backlog for Sprint 7 to determine how best to handle. We're setting up the HTML to be very piece meal, but are not determining how forms are handled as a set of input fields. 

Sound good?

Comment (maya) [inline _elements/inputs.md]: Sounds good! As long as we have it documented somewhere.

Comment (juliaelman) [inline _elements/inputs.md]: @maya added in Sprint 7 here: https://trello.com/c/MNUwuyRr/391-wrapping-all-form-fields-label-input

File: _elements/inputs.md [hunk 2]
```diff
@@ -19,19 +19,25 @@ title: Inputs
...
   <input id="input-type-text" name="input-type-text" type="text">
 
   <label for="input-focus">Text Input Focused</label>
   <input class="usa-input-focus" id="input-focus" name="input-focus" type="text">
 
-  <label for="input-error">Text Input Error</label>
-  <input class="usa-input-error" id="input-error" name="input-error" type="text">
+  <div class="input-error">
+    <label for="input-error">Text Input Error</label>
+    <span class="input-error_message" role="alert">Helpful error message</span>
```
Comment (maya) [inline _elements/inputs.md]: Shouldn't it be `input-error-message` (with a dash) since it's a different element below `input-error`?

Comment (maya) [inline _elements/inputs.md]: Please add `usa-` prefix to all classes.

Comment (juliaelman) [inline _elements/inputs.md]: Updated here b5413537212bea4a9a0504dd50f5abd1c865f6b3

**Comment by maya** [comment]
lgtm :+1:

**Pull request merged**